timer/tablestore/sql.go

```diff
@@ -0,0 +1,302 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tablestore
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/timer/api"
+)
+
+// CreateTimerTableSQL returns a SQL to create timer table
+func CreateTimerTableSQL(dbName, tableName string) string {
```

Comment:
This table is not added to the `bootstrap` stage in this PR, right?

Reply:
yes, it will be added to bootstrap if some feature want to use it